R/dsl-distributions.R

```diff
@@ -1,3 +1,29 @@
+##' Report information about supported distributions in the dsl.  This
+##' is primarily intended for use in packages which use
+##' [mcstate_dsl_parse_distribution], as this function reports
+##' information about which distributions and arguments would succeed
+##' there.
+##'
+##' @title Information about supported distributions
+##'
+##' @return A data.frame with columns
+##'
+##' * `name` the name of the distribution; each name begins with a
+##'   capital letter, and there are duplicate names where different
+##'   parametrisations are supported.
```

Comment:
```suggestion
##'   parameterisations are supported.
```

Reply:
hah, I googled that and found *four* variants. I don't have strong feelings though